CorsixTH/Lua/game_ui.lua

```diff
@@ -100,7 +100,7 @@ function GameUI:GameUI(app, local_hospital, map_editor)
   local temp_table = {}
   local shift_scroll_key_index = 1
   if type(self.app.hotkeys["ingame_scroll_shift"]) == "string" then
-    temp_table = shallow_clone({self.app.hotkeys["ingame_scroll_shift"]})
+    temp_table[#temp_table+1] = self.app.hotkeys["ingame_scroll_shift"]
```

Comment:
the index is always 1, no reason to take the length. You could also assign the table here, like you do below for lifetime symmetry. e.g. temp_table = {self.app.hotkeys["ingame_scroll_shift"]}

Reply:
Okay, changed it to 
```lua
if type(self.app.hotkeys["ingame_scroll_shift"]) == "string" then
    temp_table = {self.app.hotkeys["ingame_scroll_shift"]}
```